src/main/java/software/amazon/event/kafkaconnector/offloading/EventBridgeEventDetailValueOffloadingResult.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.offloading;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.event.kafkaconnector.EventBridgeResult;
+import software.amazon.event.kafkaconnector.util.MappedSinkRecord;
+
+public class EventBridgeEventDetailValueOffloadingResult {
```

Comment:
nit(readability): IMHO we can change these class names to just EventBridgeEventOffloading..., i.e., drop the DetailValue part and make this clear in the code comments that we only consider $.detail.value (and children) as candidates for offloading. Strictly speaking, we are never going to offload the full event detail ($.detail) because it always is set with certain Kafka keys.

Reply:
+1 for readability